numba/core/typing/typeof.py

```diff
@@ -11,6 +11,12 @@
 # terminal color markup
 _termcolor = errors.termcolor()
 
+# https://github.com/numba/numba/pull/7900#issuecomment-1065026619
+try:
+    from numpy.random._bit_generator import BitGenerator
+except ImportError:
+    from numpy.random.bit_generator import BitGenerator
```

Comment:
Can this perhaps be more explicit and based on NumPy version via `numba.np.numpy_support.numpy_version`? That way as older NumPy versions are dropped the split is likely to be found when grepping for a NumPy version tuple and the split path can be removed.